jobs/search-term-data-validation/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7
```

Comment:
This is a fairly old version of Python. the Nightly job is targeting python:3 with no minor version specified. That means it targets python 3.10 (according to https://hub.docker.com/_/python).

I would suggest using a newer version of python